components/resources/brave_components_strings.grd

```diff
@@ -524,6 +522,13 @@
       <message name="IDS_BRAVE_UI_SHOW_ALL" desc="">Show All</message>
       <message name="IDS_BRAVE_UI_SUPPORTED_SITES" desc="">Supported Sites</message>
 
+      <message name="IDS_BRAVE_UI_GRANT_FINISH_TOKEN_ADS" desc="Token text on final grant screen for UGP">Your Brave Ads Token Grant</message>
+      <message name="IDS_BRAVE_UI_GRANT_FINISH_TEXT_ADS" desc="Text on final grant screen for UGP">Your rewards grant from Brave Ads is on its way</message>
```

Comment:
needs a full-stop to be consistent with UGP

Reply:
done